### ISSUE TITLE
Boss-related inconsistencies

### ISSUE THREAD

**Issue by Subc2** [nxengine/nxengine-evo#142]
In NXEngine-evo fights with some bosses differ slightly from Cave Story experience:
- When fighting Balfrog, the little frogs (Puchi) don't hurt you when they aren't moving.
- The missiles hitting Curly (as a boss) should explode (without doing damage).
- When fighting The Sisters there should be more blood and in the ending animation they should have opened eyes.
- Ma Pignon should be immune to missiles.
- When fighting Misery using Spur or Fireball, there should be more blood.
- When you aren't moving, the spikes appearing just before Ballos's final form don't hurt you.

**Comment by xordspar0** [comment]
> in the ending animation they should have opened eyes.

You're right. Caitlin Shaw thought they had their eyes closed, but I guess she was wrong about that.

https://github.com/nxengine/nxengine-evo/blob/56fef6993e410b97346a7866d8d02e4655077fa5/ai/boss/sisters.cpp#L34

I guess the closed eyes animation frame is never used then?
